airbyte-integrations/connectors/source-google-search-console/source_google_search_console/config_migrations.py

```diff
@@ -77,7 +77,7 @@ def emit_control_message(cls, migrated_config: Mapping[str, Any]) -> None:
         cls.message_repository.emit_message(create_connector_config_control_message(migrated_config))
         # emit the Airbyte Control Message from message queue to stdout
         for message in cls.message_repository._message_queue:
-            print(message.json(exclude_unset=True))
+            print(message)
```

Comment:
This seems odd to me as the AirbyteMessage object even with the new dataclass implementation will probably not be represented as JSON. Shouldn't it be something like `print(orjson.dumps(AirbyteMessageSerializer.dump(message)).decode())`?